edge-hub/docker/linux/amd64/Dockerfile

```diff
@@ -24,10 +24,11 @@ WORKDIR /app
 
 COPY $EXE_DIR/ ./	
 
-# Expose MQTT, AMQP and HTTPS ports	
+# Expose MQTT, AMQP, HTTPS, and metrics ports
 EXPOSE 8883/tcp	
 EXPOSE 5671/tcp	
 EXPOSE 443/tcp	
+EXPOSE 9600/tcp
```

Comment:
if customer runs metric module in iotedge, this port is not required to expose.  Therefore you should not add this line.

Reply:
I brought this point up with Anca and we decided this was fine because expose does not actually expose the port, but allows the port to be exposed with CreateOptions.